kythe/typescript/testdata/tsx.tsx

```diff
@@ -0,0 +1,11 @@
+// Test TypeScript JSX
+
+//- @#0"value" defines/binding Value
+const value = 'value';
+function render() {
+  return (
+    //- @#0"value" defines/binding vname("render.value", _, _, _, _)
```

Comment:
This is weird, how is this defining a binding?

Reply:
Because this "value" is now a property on the element returned by this function, so we can treat it as variable-like.

Reply:
Won't this conflict with a local named 'value' within the render function?

Reply:
Yes. You're totally right. Fixed.

This supports a very limited subset of tsx, but i don't think we should support much other than what customers downstream use, which is what's been added here.